helm/_mailpit/values.yaml

```diff
@@ -73,3 +77,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+mailpit:
+  dbPath: /var/lib/mailpit/mailpit.db
+
+
+persistence:
```

Comment:
no need to introduce a new attribute, just use `volumes` and revert `volumes` back in deployment template